controllers/managedocs_controller.go

```diff
@@ -131,7 +132,10 @@ func (r *ManagedOCSReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 	}
 }
 
-func (r *ManagedOCSReconciler) reconcilePhases() error {
+func (r *ManagedOCSReconciler) reconcilePhases(req ctrl.Request) error {
```

Comment:
The `req` param was removed intentionally in the last commit. Your only use for it is to get access to the namespace, for that you can now use `r.namespace` instead